examples/proc_FIR.py

```diff
@@ -50,9 +49,8 @@
         indirect = "vd"
         direct = "t2"
         if clock_correction:
-            # {{{ clock correction
-            s = prscr.clock_correct(s, fl=fl)
-            # }}}
+            s = prscr.clock_correct(s)
+        # TODO ☐: why do you return dropped?
```

Comment:
just address -- to me, it seems like `s.squeeze()` should be sufficient.

Reply:
I had `s.squeeze()` but you were suprised this worked. I think `s = s.squeeze()` is the correct way to do this. I returned to list of dropped dimmensions as I wasn't sure if this was what you wanted by 'implement pyspecdata module'